src/sentry/integrations/gitlab/client.py

```diff
@@ -134,31 +135,53 @@ def get_user(self):
         """
         return self.get(GitLabApiClientPath.user)
 
-    def search_group_projects(self, group, query=None, simple=True):
-        """Get projects for a group
+    def search_projects(self, group=None, query=None, simple=True):
+        """Get projects
 
         See https://docs.gitlab.com/ee/api/groups.html#list-a-group-s-projects
+        and https://docs.gitlab.com/ee/api/projects.html#list-all-projects
         """
 
         def gen_params(page_number, page_size):
-            # simple param returns limited fields for the project.
+            # Simple param returns limited fields for the project.
             # Really useful, because we often don't need most of the project information
-            return {
+            params = {
                 "search": query,
                 "simple": simple,
-                "include_subgroups": self.metadata.get("include_subgroups", False),
+                "order_by": "last_activity_at",
                 "page": page_number + 1,  # page starts at 1
                 "per_page": page_size,
             }
+            if group:
+                params.update(
+                    {
+                        "include_subgroups": self.metadata.get("include_subgroups", False),
+                    }
+                )
+            else:
+                params.update(
+                    {
+                        "membership": True,
```

Comment:
I added this because for the [project endpoint](https://docs.gitlab.com/ee/api/projects.html#list-all-projects), without it you'll just see every public GitLab repo in the repositories tab (other people's public repos).